experimental/genericTemplate.json

```diff
@@ -20,9 +20,19 @@
             "body": [
                 {
                     "type": "logStatement",
-                    "attributes": ["hello world"]
+                    "args": ["hello world"]
                 }
             ]   
         }
+    },
+    "customStatementNoArgs": {
+        "py": "custom_function_name()",
```

Comment:
The way I was imagining it was that the generic tree has no knowlede about python or js, but that each language converts to that format and prints from that format. 

In this case you would store the statement name in some chosen casing maybe camel case and then python would convert to that and so would js. Then when going from gast -> python you  would again convert to the format of that language.

Reply:
yeah- agreed! we were honestly just using the generic template as a way to coordinate efforts across python and javascript. That way, @swalsh15 would know how to format conversion from jsAst to gast in the same way that I do it from pyAst. The language specific stuff was mostly to make it more clear for us in case of ambiguity in the name of "customStatementNoArgs"